### DISCUSSION TITLE
zksync not functioning

### DISCUSSION THREAD

**Discussion by EarlEdalok** [Cyfrin/foundry-full-course-cu#3073] [General]
hi, cyfrin co.
![Screenshot 2024-12-19 112019](https://github.com/user-attachments/assets/f37ea13d-cbf2-47e7-b544-530e318de1c0)
 the link to the zksync keeps on loading for a long time. Any solution?

**Comment by EngrPips** [comment]
Hello @EarlEdalok, Please clear your browser cache and then retry.
